packages/core/src/database/postgres/service.test.ts

```diff
@@ -63,15 +63,19 @@ describe.skipIf(shouldSkip)("postgres database", () => {
     });
 
     // Cache database, metadata tables, and cache tables were created
-    expect(await getTableNames(database.db, "ponder_cache")).toStrictEqual([
-      "kysely_migration",
-      "kysely_migration_lock",
-      "function_metadata",
-      "table_metadata",
-      "instance_metadata",
-      "0xPet",
-      "0xPerson",
-    ]);
+    expect(
+      new Set(await getTableNames(database.db, "ponder_cache")),
```

Comment:
Could also do `.sort()` on both arrays here. Unfortunately, don't think vitest has a matcher for arrays ignoring order.